elf/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(elf)
+
+set(toStringInputHeader ${CMAKE_CURRENT_LIST_DIR}/data.hh)
+set(to_string_src ${CMAKE_CURRENT_BINARY_DIR}/to_string.cc)
+set(configured_to_string_script ${CMAKE_CURRENT_BINARY_DIR}/create_to_string_code.sh)
+
+configure_file(create_to_string_code.sh ${configured_to_string_script})
+
+add_custom_command(
+  OUTPUT ${to_string_src}
```

Comment:
Same remark as the dwarf library.
I would not use bash but add these commands directly here.
See https://github.com/conan-io/conan-center-index/pull/3283 again